src/app.test.ts

```diff
@@ -419,6 +448,35 @@ test('POST /styles when providing valid style returns resource with id and alter
   })
 })
 
+test('POST /styles when providing valid style creates styles for its raster sources', async (t) => {
```

Comment:
Is this desired behaviour? I remember discussing this but I don't remember the conclusion and I'm not sure if this makes sense?

Reply:
Good question - I'm not entirely sure! Think the work from https://github.com/digidem/mapeo-mapserver/pull/25 resulted in this being a behavior at the moment. Wrote a test for it just to get a better idea of what was going on but probably would be helpful to discuss.

When creating the style, we create the offline tilesets based on its `sources` field, which in turn may cause the raster styles to be generated for the sources. is that flow doing too much?

Reply:
addressed via https://github.com/digidem/mapeo-mapserver/pull/19/commits/e3f9906515c62b2f943b368fe3e2745eff95624b